api/query/query.go

```diff
@@ -61,6 +61,30 @@ func (qh queryHandler) processInput(w http.ResponseWriter, r *http.Request) (*sh
 	return &filters, testRuns, summaries, nil
 }
 
+func (qh queryHandler) getVersionFiles(w http.ResponseWriter, r *http.Request) ([]string, error) {
+
+	filters, err := shared.ParseQueryFilterParams(r.URL.Query())
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return nil, err
+	}
+	testRuns, filters, err := qh.getRunsAndFilters(filters)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+		return nil, err
+	}
+	versionFiles := make([]string, 0, len(testRuns))
+	for _, testRun := range testRuns {
+		summaryURL := shared.GetResultsURL(testRun, "")
+		url := summaryURL[:len(summaryURL)-16] + "_version.txt"
```

Comment:
could you make 16 a constant and a comment about it please? i'm trying to learn about this offset.

Reply:
Sure, I'll update this to explain what is going on here. It's unintuitive (and probably not the best approach), but it definitely deserves an explanation 🙂